pipelines/cellpainting/cpd_analysis_pipeline.wdl

```diff
@@ -15,6 +15,8 @@ workflow cpd_analysis_pipeline {
     # Specify input file information
     String images_directory_gsurl
     String? file_extension = ".tiff"
+    File load_data_csv
+    String illum_directory_gsurl = "${images_directory}/illum"
```

Comment:
Maybe make this optional, like
```wdl
String? illum_directory_gsurl = "${images_directory}/illum"
```


You may also need to add a line after line 31 (outside the `input` block) like this
```wdl
String illum_directory = sub(illum_directory_gsurl, "/+$", "")
```
and then use `illum_directory` in line 45.  Since this is an input, then the user could change it and have a trailing slash by mistake.

Reply:
Oh yeah, it seems you're right, I didn't realize that. `file_extension`, which is a `String?`, is listed as an optional input, and then `splitby_metadata`, which is a `String` is also listed as optional because it has a default value in the WDL.  Huh!

Reply:
added the line to remove the slash